package.json

```diff
@@ -44,5 +46,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "type": "module"
```

Comment:
Had to add this to make it explicit that we were using import/export rather than require. TS was complaining